interface/ceed.c

```diff
@@ -379,6 +382,26 @@ int CeedInit(const char *resource, Ceed *ceed) {
   return 0;
 }
 
+/**
+  @brief Retrieve a parent CEED
+
+  @param ceed           Ceed to retrieve parent of
+  @param[out] parent    Address to save the parent to
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Developer
+**/
+int CeedGetParent(Ceed ceed, Ceed *parent) {
```

Comment:
Added to roll back up the list of delegates. This lets us use a delegated CeedBasis and specific CeedTensorContract